src/web/handlers/PanGestureHandler.ts

```diff
@@ -52,7 +52,7 @@ export default class PanGestureHandler extends GestureHandler {
   private lastY = 0;
 
   private activateAfterLongPress = 0;
-  private activationTimeout = 0;
+  private activationTimeout: NodeJS.Timeout | undefined;
```

Comment:
```suggestion
  private activationTimeout: ReturnType<typeof setTimeout> | undefined;
```

Reply:
Done in [3f244d2](https://github.com/software-mansion/react-native-gesture-handler/pull/2852/commits/3f244d27a397c68bf80abedc2c9ed1e450ebf534)